cs-vmware/src/main/java/io/cloudslang/content/vmware/services/DeployOvfTemplateService.java

```diff
@@ -0,0 +1,264 @@
+package io.cloudslang.content.vmware.services;
+
+import com.vmware.vim25.HttpNfcLeaseDeviceUrl;
+import com.vmware.vim25.HttpNfcLeaseInfo;
+import com.vmware.vim25.InvalidPropertyFaultMsg;
```

Comment:
Ok, did that on entire cs-vmware project. Note that the number of changed files of the pull request will increase by @30 files. 😃